### PR TITLE
feature: add restart skeleton

### PR DESCRIPTION
**Pull request by Letty5411**
Signed-off-by: letty <letty.ll@alibaba-inc.com>

### CODE REVIEW

**Comment by HusterWan** [comment]
conflicted,  please rebase you code ,thanks :) @Letty5411

File: cli/restart.go [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+// addFlags adds flags for specific command.
+func (rc *RestartCommand) addFlags() {
+	flagSet := rc.cmd.Flags()
+	flagSet.IntVarP(&rc.time, "time", "t", 10,
+		"Seconds to wait for stop before killing the container (default 10)")
+}
+
+// runrestart is the entry of restart command.
+func (rc *RestartCommand) runrestart(args []string) error {
```
Comment (HusterWan) [inline cli/restart.go]: s/runrestart/runRestart

File: cli/restart.go [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+	"github.com/spf13/cobra"
+)
+
+// restartDescription is used to describe restart command in detail and auto generate command doc.
+var restartDescription = "restart one or more containers"
+
+// restartCommand uses to implement 'restart' command, it restarts a container.
+type RestartCommand struct {
+	baseCommand
+	time int
```
Comment (HusterWan) [inline cli/restart.go]: s/time/timeout will be more reasonable, WDYT?

**Comment by HusterWan** [comment]
@Letty5411 Maybe we also need add a skeleton of ci test, WDYT?

File: cli/restart.go (hunk 2 above)
Comment (Letty5411) [reply cli/restart.go]: ok~

File: cli/restart.go [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+	"context"
+
+	"fmt"
+	"github.com/spf13/cobra"
+)
+
+// restartDescription is used to describe restart command in detail and auto generate command doc.
+var restartDescription = "restart one or more containers"
+
+// restartCommand uses to implement 'restart' command, it restarts a container.
```
Comment (HusterWan) [inline cli/restart.go]: s/restartCommand/RestartCommand
BTW, as your restartDescription says `restart one or more containers`, so this comment should change

**Comment by HusterWan** [comment]
LGTM

**Comment by HusterWan** [comment]
This PR is not the reason CI failed, so I will merge this PR

**Pull request merged**